extension/app/src/lib/storage.ts

```diff
@@ -208,3 +208,28 @@ export const getFileContentFragmentId = async (
   const result = await chrome.storage.local.get([key]);
   return result[key] ?? null;
 };
+
+export type SavedAssistantConfiguration = {
```

Comment:
NIT: `AssistantConfiguration` could easily be mislead with `AgentConfiguration` so I would be in favor of renaming for clarity. What about `QuickAction` or event `ContextMenuActions` to be super explicit?